### ISSUE TITLE
subjectOrientation definitions are somewhat circular

### ISSUE THREAD

**Issue by baskaufs** [tdwg/ac#239]
The current subjectOrientation definitions are somewhat circular, giving the label as part of the definition. Perhaps we need to use the ontology definitions instead.

This is particularly apparent with the new bryophyte orientations where dorsal and ventral are being used with non-bilaterally symnetric organism parts.

**Comment by baskaufs** [comment]
I have made an attempt to improve this [here](https://github.com/tdwg/ac/blob/cec167d4fc85b5edaa2559572e5d1c060e4dcb3a/views/code/subjectOrientation/subjectOrientation_cv.csv). The ontology definitions do not help -- they are equally circular and in some cases don't say much at all. However, it's nearly impossible to define left and right, and dorsal and ventral without being somewhat circular or having a really complicated definition. 

I decided to change the label of `adaxial side` and `abaxial side` to `upper side` and `lower side` respectively. That's a less technical label, but the technical details are in the definition (including "adaxial" and "abaxial"). One thing I noticed when testing was that it's really easy to make a mistake and switch up "adaxial" and "abaxial", either by thinking backwards or just mis-typing it into a spreadsheet. I think that's less likely with "upper" and "lower". I also added a usage note saying to use upper and lower instead of the `dorsal` and `ventral` terms used with bilaterally symmetric animals. I think that makes sense since if someone was searching for views of plants looking down from the "top", they would rather get liverwort images with herbaceous plants than with fish or mammals.

I also stated in the usage instructions that `upper side` and `lower side` should be used for horizontal parts of plants or fungi that don't have a central axis. The rationale here is the same: people looking for pictures of any plant or fungi would probably prefer to see all parts whose side points away from the ground together, rather than seeing them with the backs of birds, lizards, etc.
